core/msgio/android.go

```diff
@@ -28,6 +30,9 @@ func SyncAndroidChannel(ctx context.Context, fc FCMClient, channel *models.Chann
 
 	// no FCM ID for this channel, noop, we can't trigger a sync
 	fcmID := channel.ConfigValue(models.ChannelConfigFCMID, "")
+	if registrationID != "" {
```

Comment:
needed for released channel since we clear the FCM ID on them but need one last sync

Reply:
Is there a good reason we do that? Seems weird to lose that information.

Reply:
I guess that has to do we making sure a released channel will not sync messages and send them any more

Reply:
But that can be determined by `Channel.is_active`.. I don't think we need to discard the FCM ID

Reply:
Or that could be a recommendation by firebase so we do not use that ID anymore in case that could affect the system with unnecessary traffic

And that got be related to the recommendation to verify the token IDs too
https://firebase.google.com/docs/auth/admin/verify-id-tokens#go

Reply:
But why would we use the id again? The channel is being soft-deleted... it no longer exists to the syncing code.

Reply:
Well, we can remove that for now, until we remember why that was added.

We'll need to check the channel release on rapidpro to not remove the FCM ID on the channel config

Reply:
https://github.com/nyaruka/rapidpro/pull/5367